rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/330_fetch_fields.yml

```diff
@@ -1064,3 +1064,65 @@ test fetching metadata fields:
   - match: { hits.hits.0.fields._id.0: "1" }
   - match: { hits.hits.0.fields._index.0: "test" }
   - match: { hits.hits.0.fields._version.0: 1 }
+
+---
+fetch geo_point:
```

Comment:
You can run this in two ways:
```
./gradlew -p rest-api-spec/ check -Dtests.method='*fetch_fields*'
./gradlew -p x-pack/qa/runtime-fields/core-with-mapped/ check -Dtests.method='*fetch_fields*'
```

The former runs against indexed fields and passes without this change. The latter runs against runtime fields that load from `_source` and fails without this change.